CIP-AFFIRM/README.md

```diff
@@ -0,0 +1,126 @@
+---
+CIP: ?
+Title: Affirmation Graph
+Category: Tools
+Status: Proposed
+Authors:
+  - Kieran Simkin <hi@clg.wtf>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/938
+    - https://forum.cardano.org/t/introducing-the-affirmation-graph-request-for-comment/138320
+Created: 2024-10-26
+License: CC-BY-4.0
+---
+
+## Abstract
+We need a truly decentralized alternative to KYC for when we need to establish the authenticity of a wallet or make some judgement about its trustworthiness in a systematic and repeatable manner. This CIP defines a mechanism whereby any member of the community can publicly vouch for the authenticity of a wallet by submitting an "affirmation" transaction, it also provides a mechanism to revoke such an affirmation, if, for example a wallet becomes compromised, or its owner turns out to be a charlatan. 
+It is consequently possible to observe the current UTxO set and draw an "Affirmation Graph", a multidigraph representing the trust relationships between wallets - we can then use an algorithm to assign trust rankings and decide whether or not to trust an unknown wallet.
+
+![Affirmation Graph example](graph.svg)
+
+## Motivation: why is this CIP necessary?
+When transacting with an unknown person, currently if we want to make sure we're not dealing with a bot or a grifter, we have to perform our own detective work - examining their current token holdings and transaction history, then we make some informed guess about what we're dealing with. This is both laborious and inconsistent. This CIP aims to provide a better alternative which is repeatable and automatable, this could drastically reduce the risk of fraud and accidental loss of funds.
```

Comment:
The claim that this CIP could "drastically reduce the risk of fraud and accidental loss of funds" feels overstated. While it makes trust evaluations more systematic and repeatable, it does not eliminate risks like system manipulation or false trust. A more balanced phrasing might highlight that it complements manual diligence rather than fully replacing it.

Something like

*"This CIP proposes a decentralized mechanism for assessing wallet trustworthiness in a systematic and repeatable way, aiming to reduce the effort and inconsistencies of manual evaluations. While it does not eliminate risk entirely, it provides a new tool to complement existing approaches to fraud prevention."*

Reply:
I'll take that wording :) Thanks.